specification/machinelearningservices/data-plane/readme.md

```diff
@@ -31,6 +31,20 @@ use-internal-constructors: true
 add-credentials: true
 ```
 
+## Supression
+
+``` yaml
+directive:
+  - suppress: AvoidNestedProperties
```

Comment:
Please take a look and follow https://github.com/Azure/adx-documentation-pr/wiki/Swagger-Validation-Errors-Suppression for suppression. Thanks!